### ISSUE TITLE
[BUG] phrase git_lab_sync history = ERROR: json: cannot unmarshal string into Go struct field GitlabSyncHistory.status 

### ISSUE THREAD

**Issue by horstmannmat** [phrase/openapi#362]
When executing the `phrase git_lab_sync history` in the latest version of the phrase-cli, it raises an error message:
```bash
ERROR: json: cannot unmarshal string into Go struct field GitlabSyncHistory.status of type int32 
```
Which seems to be related to a type mismatch between two models, specifically [gitlab_sync.yaml#L28-L29](https://github.com/phrase/openapi/blob/master/schemas/gitlab_sync.yaml#L28-L29) and [gitlab_sync_history.yaml#L6-L7](https://github.com/phrase/openapi/blob/master/schemas/gitlab_sync_history.yaml#L6-L7).

This discrepancy in types causes the command to fail and prevents the expected functionality.

<table>
<tr>
  <th>gitlab_sync.yaml</th><th>gitlab_sync_history.yaml</th>
</tr>

<tr>
  <td>
    <pre>
last_status:
  type: string
    </pre>
  </td>
  <td>
    <pre>
status:
  type: integer
    </pre>
  </td>
</tr>
</table>

**Comment by horstmannmat** [comment]
I will create a quick PR for it, but I lack ways of testing it properly

**Comment by theSoenke** [comment]
Hi @horstmannmat, this should now be resolved in the latest release 2.8.3. Thanks again for the report!
